formats/json/jsMain/src/kotlinx/serialization/json/Dynamics.kt

```diff
@@ -45,11 +45,10 @@ public inline fun <reified T> Json.decodeFromDynamic(dynamic: dynamic): T =
     decodeFromDynamic(serializersModule.serializer(), dynamic)
 
 /**
- * Converts Kotlin data structures to plain Javascript objects
+ * Converts Kotlin data structures to plain Javascript objects, including support for polymorphism
```

Comment:
```suggestion
 * Converts Kotlin data structures to plain Javascript objects
```

There's no need to emphasise that it supports functionality that should have been there by design — removing 'limitation' line is enough